internal/web/templates/pages/package.html

```diff
@@ -72,6 +72,16 @@ <h2 class="text-reset">Package Details</h2>
             {{ .Manifest.LongDescription }}
           {{ end }}
         </span>
+        <br />
+        <span>
+          <i>
+            Latest Verrsion: {{ .LatestVersion }}
+          </i>
+          <br />
+          {{ if ne .Status nil }}
```

Comment:
why are you checking whether `.Status` is not nil? inside the `if` block the `.Status` is not being accessed at all. or am i missing something?